src/actors/builtin/init/init_actor.go

```diff
@@ -61,7 +62,8 @@ func (a *InitActorCode_I) Exec(rt Runtime, execCodeID abi.ActorCodeID, construct
 	})
 
 	return rt.ValueReturn(
-		Bytes(addr.Serialize_Address_Compact(idAddr)))
+		Bytes(idAddr.Bytes()),
```

Comment:
nit: idAddr.Bytes() should suffice

Reply:
Actually this should be the CBOR serialization.